### PR TITLE
feat(db): implement base db functions

### PR DESCRIPTION
**Pull request by romutchio**

### CODE REVIEW

File: src/main/scala/database/DatabaseClient.scala [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "jdbc:postgresql://localhost:5432/cavisto",
+    "cavisto",
+    "cavisto"
+  )
+
+  implicit val metaAdviseState: Meta[AdviseState] = new Meta(pgDecoderGet, pgEncoderPut)
+
+  def dropUsersTable(): F[Int] = sql"""DROP TABLE IF EXISTS users""".update.run.transact(transactor)
+
+  def createUsersTable(): F[Int] =
```
Comment (ki4kis) [inline src/main/scala/database/DatabaseClient.scala]: Table creation should happen outside the app.
You can make an `.sql` file that you can run when setting up the DB.

File: src/main/scala/Main.scala [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
-      _ = println(resp1)
-      _ = println(resp2)
+      _ <- databaseClient.dropAdviseHistoryTable() >> databaseClient.dropUsersTable()
+      _ <- databaseClient.createUsersTable() >> databaseClient.createAdviseHistoryTable()
+      userId <- databaseClient.insertUser(100, Some("romutchio"))
+      adviseId <- databaseClient.insertAdviseHistory(userId, AdviseState(Some("France"), Some("Red"), Some(3), Some(20)))
+      user <- databaseClient.selectUser(100)
+      advise <- databaseClient.getAdviseHistory(adviseId)
+      _ = println(user)
+      _ = println(advise)
```
Comment (ki4kis) [inline src/main/scala/Main.scala]: You should do this kind of stuff in a test :)

Comment (romutchio) [reply src/main/scala/Main.scala]: it's kinda demonstration, for running and debugging

File: docker-compose.yml
Comment (ki4kis) [inline docker-compose.yml:1]: Mount the data folder of postgres. You won't need to recreate the database data each time you run the docker

**Pull request merged**